articles/pre-deployment/index.html

```diff
@@ -39,6 +40,12 @@ <h1>Pre-Deployment Tests</h1>
       </li>
     </ul>
   </li>
+  <li>
```

Comment:
This doc is already listed (see the first link in this page). If for some reason we feel the need to emphasize this paragraph (although this is not a practice we use in our docs) then it should be added under the first one (like the `Required Fixes` link is listed under the `Production Check Results`).

(Although the right solution to me is to make this a doc on its own)

Reply:
Can we replace the html file with an md file or will that break something?

Reply:
The request was to make all the sections of the how-to-run-test page easily navigable from the landing page so that it reflects the actual doc structure/gives the user a glimpse of all the content. (Which is why the "How to Read Your Results Set" link was also added below.) You're right, though; with the new IA and content type, I do see the plan being to break this doc into pieces. Too much info is buried from the user right now.